liminal/runners/airflow/tasks/python.py

```diff
@@ -15,131 +15,118 @@
         self.kubernetes_kwargs = self.__kubernetes_kwargs()
         self.cmds, self.arguments = self.__kubernetes_cmds_and_arguments()
-        self.input_task_id = self.task_name + '_input'
-        self.executors = self.__executors()
+        self.parallelism = self.__parallelism()
 
     def apply_task_to_dag(self):
-        input_task = None
+        if self.parallelism == 1:
+            return self.__apply_task_to_dag_single_executor()
+        else:
+            return self.__apply_task_to_dag_multiple_executors()
 
-        if self.input_type in ['static', 'task']:
-            input_task = self.__input_task()
+    def _volumes_config(self):
+        volumes_config = self.liminal_config.get('volumes', [])
 
-        if self.executors == 1:
-            return self.__apply_task_to_dag_single_executor(input_task)
-        else:
-            return self.__apply_task_to_dag_multiple_executors(input_task)
+        for volume in volumes_config:
+            if 'local' in volume:
+                volume['persistentVolumeClaim'] = {
+                    'claimName': f"{volume['volume']}-pvc"
+                }
 
-    def __apply_task_to_dag_multiple_executors(self, input_task):
-        if not input_task:
-            input_task = DummyOperator(
-                task_id=self.input_task_id,
-                trigger_rule=self.trigger_rule,
-                dag=self.dag
-            )
+        return volumes_config
+
+    def __apply_task_to_dag_multiple_executors(self):
+        start_task = DummyOperator(
+            task_id=f'{self.task_name}_parallelize',
+            trigger_rule=self.trigger_rule,
+            dag=self.dag
+        )
 
         end_task = DummyOperator(
             task_id=self.task_name,
             dag=self.dag
         )
 
         if self.parent:
-            self.parent.set_downstream(input_task)
+            self.parent.set_downstream(start_task)
 
-            for i in range(self.executors):
+            for i in range(self.parallelism):
                 split_task = self.__create_pod_operator(
-                    task_id=f'''{self.task_name}_{i}''',
-                    task_split=i,
-                    image=self.image
+                    image=self.image,
+                    task_id=i
                 )
 
-                input_task.set_downstream(split_task)
+                start_task.set_downstream(split_task)
 
                 split_task.set_downstream(end_task)
 
         return end_task
 
-    def __create_pod_operator(self, task_id, task_split, image):
-        return KubernetesPodOperatorWithInputAndOutput(
-            task_id=task_id,
-            input_task_id=self.input_task_id,
-            task_split=task_split if task_split else 0,
+    def __create_pod_operator(self, image: str, task_id: Optional[int] = None):
+        env_vars = self.env_vars
+
+        if task_id is not None:
+            env_vars = self.env_vars.copy()
+            env_vars['LIMINAL_SPLIT_ID'] = str(task_id)
```

Comment:
We need it.
It does have to do with executors rather than volumes but I added it in this PR so I can test functionality I wanted in `test_python_image_builder`